### ISSUE TITLE
Sent messages are showing as user status

### ISSUE THREAD

**Issue by dogcalas** [open-wa/wa-automate-nodejs#393]
**Describe the bug**
Some messages sent to users are also showing as status.

**Steps to Reproduce**
I am only sending messages.

**create() code**
A simple sendtext.
```javascript
        const codenumber = message.from.split('@')[0].substr(0, 3)
        if (codenumber != '593') {
            await globalClient.sendText(message.from, bot.chat('noec'))
        }
```

**Expected behavior**
If I am not sending anything to status, not new status should appear. This is happening, not only with one message, others sent messages eventually appears in status.

**DEBUG INFO**
This is the info printed to the console when you start your app. It should look like this
```
Debug Info {
  WA_VERSION: 2.2017.6,
  PAGE_UA: WhatsApp/0.4.613 Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.88 Safari/537.36,
  WA_AUTOMATE_VERSION: 1.9.4,
  BROWSER_VERSION: HeadlessChrome/81.0.4044.129
}
```

**Host (please complete the following information):**
 - Ubuntu 18.04

**Comment by smashah** [comment]
@dogcalas can you use the latest version and provide screenshots please

**Comment by dogcalas** [comment]
Maybe is ridiculous, but the only change I made since this is happening, is update open-wa  version (as you recomend in readme). How is possible that the messages start appearing in the status, if I don't  change my code and the bot is in production for more than one month.

This is partialy how I manage the messages.
```javascript
const { ev, create } = require('@open-wa/wa-automate');
const { default: PQueue } = require("p-queue");
const queue = new PQueue({
  concurrency: 1,
  autoStart: false
});
let globalClient;


const proc = async message => {
  try {
    if (message.type == "location") {
      await globalClient.sendText(message.from, bot.chat('cotizacion'))
    } else if (message.type == 'chat') {
      bot.handleWit(message, globalClient)
    }
  } catch (err) {
    console.log(err)
  }
  return true;
}

const processMessage = message => queue.add(proc(message));

async function start(client) {
  try {
    globalClient = client;

    client.onStateChanged(state => {
      console.log('statechanged', state)
      if (state === "CONFLICT") client.forceRefocus();
    });

    client.onAck((msg) => () => { })
    await client.onMessage(processMessage);
    queue.start();

  } catch (err) {
    console.log(err)
    process.exit(1)
  }

}

///////////////////////// Create Whatsapp Session in Chrome //////////////////////////
function startSession() {
  create('session',
    {
      // executablePath: 'chromium-browser',
      headless: false,
      executablePath: '/usr/bin/google-chrome',
      // devtools: true,
      qrRefreshS: 30,
      // killTimer: 40,
      autoRefresh: true,
      // useChrome: true,
      throwErrorOnTosBlock: true,
    }
  ).then(client => start(client))
    .catch((err) => {
      console.log("Ha ocurrido un error :: ", err)
      process.exit(1);
    });
}
startSession();
```
The method ``` bot.handleWit(message, globalClient) ``` process the message and write an answer with ```sendText``` or ```sendFile```.

**Comment by smashah** [comment]
@dogcalas I mean the problem itself is ridiculous not the fact that you made an issue for it. Can you provide a screenshot of the message and then the corresponding status.

**Comment by dogcalas** [comment]
From the Bot contact, I can see 2 status on my phone, that correspond to messages normally sent by the Bot.
![d5c416e5-ab2c-41c4-8379-2d661fbafbd8](https://user-images.githubusercontent.com/2521769/80924715-207a2f80-8d50-11ea-9537-530b79d44083.jpeg)
Also an image was published.
![31f42cff-6e06-48c9-a0d9-22a259f18d3f](https://user-images.githubusercontent.com/2521769/80924916-9f239c80-8d51-11ea-8967-f564a821bc34.jpeg)

**Comment by smashah** [comment]
You'll need to be more clear and specific about who sends what message and which messages trigger this behaviour. Are these screenshots of the bot? Or are they of your phone? Is it all messages or some? Needs more information

**Comment by smashah** [comment]
You'll need to be more clear and specific about who sends what message and which messages trigger this behaviour. Are these screenshots of the bot? Or are they of your phone? Is it all messages or some? Needs more information

**Comment by dogcalas** [comment]
This is happening only in production were a dozen of users write at the same time. The pictures are from my phone, because the whatsapp bot is running in the client's phone, and I don't have access to it right now.  
I am really upset with myself, because I cannot give you more information about this behavior. I cannot find the exact condition that make this happen.

The facts are:
1. The messages in the bot status, are messages that the bot is intended to send
2. Sent pictures by the bot also has appeared in the status
3. Problems started  after an upgrade to  1.9.x
4. The occurrence is very low, today only happened once.

Something curios is that one of the status showed was message that is sent only for other countries users. I asked to the client who is using the phone, but she didn't find  any foreign number in the contacts.

**Comment by smashah** [comment]
@dogcalas, for now, please just use this:

```javascript
  const codenumber = message.from.split('@')[0].substr(0, 3)
        if (codenumber != '593' && message.from !== 'status@broadcast') {
            await globalClient.sendText(message.from, bot.chat('noec'))
        }
```

report back if there is more info. closing for now.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.8](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.7...1.9.8) (2020-05-04)

- Release 1.9.8 [`fecde1e`](https://github.com/open-wa/wa-automate-nodejs/commit/fecde1ed4f00d3874618bfac7af87e2759d9c8c8)
- fix: Ignore status messages in onMessage #393 [`8d591ca`](https://github.com/open-wa/wa-automate-nodejs/commit/8d591caf3e8c37bbb662019a7f393b8b8b4b7b21)
- update release image [`ed5636e`](https://github.com/open-wa/wa-automate-nodejs/commit/ed5636e648dae70f8b6de74ed4512647c44ae3f2)

**Comment by smashah** [comment]
@dogcalas looking further into it. I can confirm that when a contact of the bot (who also has a chat with the bot) makes a new story on their phone, it triggers `Store.Msg.on('add',...`

This, in turn, triggers onMessage. v.1.9.8 filters out any story 'messages'.
